### ISSUE TITLE
Make force quit delete temporary files

### ISSUE THREAD

**Issue by Randomblock1** [madMAx43v3r/chia-plotter#502]
When the program is quit completely, since there's no way to resume it, I think deleting the temporary files would be a good idea, EXCEPT for a completed plot that is being moved (just delete the .tmp from the name). Plotman does this, I believe, and it would be quite nice to be able to just start a new command (maybe you make a typo and want to fix it) without having to manually remove the tmp files.

**Comment by VertiHydro** [comment]
> When the program is quit completely, since there's no way to resume it, I think deleting the temporary files would be a good idea, EXCEPT for a completed plot that is being moved (just delete the .tmp from the name). Plotman does this, I believe, and it would be quite nice to be able to just start a new command (maybe you make a typo and want to fix it) without having to manually remove the tmp files.

Well the issue is that force quitting the program stops any part of it from running immediately, including any part that could delete temporary files. Therefore, you would have to run a second program that checks the status of the first. Or, you could just save yourself the hassle and just use the command `sudo rm *.tmp` in the temp folder.

**Comment by Randomblock1** [comment]
No, when it detects a CTRL-C pressed twice, it prints a message and then exits itself. There is certainly room to add this feature before it exits. [Look here](https://github.com/madMAx43v3r/chia-plotter/commit/8d5dcae53e84bbb6421628fe460961ff9db90118#diff-5ef857c22a4861474765f7dd4e61ac3b3a892844116f9d7f86c502002490268a).
```
if ( ( (get_wall_time_micros()-interrupt_timestamp) / 1e6) &lt;= 1 ) {
    std::cout << "Double CTRL-C pressed, exiting now!!!" << std::endl;
    /* DELETE TEMP FILES HERE */
    exit(-4);
} else
    interrupt_timestamp=get_wall_time_micros();
```

**Comment by VertiHydro** [comment]
> No, when it detects a CTRL-C pressed twice, it prints a message and then exits itself. There is certainly room to add this feature before it exits. [Look here](https://github.com/madMAx43v3r/chia-plotter/commit/8d5dcae53e84bbb6421628fe460961ff9db90118#diff-5ef857c22a4861474765f7dd4e61ac3b3a892844116f9d7f86c502002490268a).
> 
> ```
> if ( ( (get_wall_time_micros()-interrupt_timestamp) / 1e6) &lt;= 1 ) {
>     std::cout << "Double CTRL-C pressed, exiting now!!!" << std::endl;
>     /* DELETE TEMP FILES HERE */
>     exit(-4);
> } else
>     interrupt_timestamp=get_wall_time_micros();
> ```

Alright, point taken. The ctrl-c functionality has been changed so many times in the last few days I can't keep up! This is a fair idea then, but don't be surprised if this isn't implemented because of a desire not to overcomplicate the plotter. It could well be useful though!

**Comment by madMAx43v3r** [comment]
Yes it's possible, I saw that feature request coming.
